### ISSUE TITLE
Accessing https://cloud.tenable.com/

### ISSUE THREAD

**Issue by AlexDBA012** [tenable/Posh-Nessus#25]
Hi

can the POSH-Nessus module be used to access and manage cloud based Nessus scans

Cheers

Ale

**Comment by riskymanag3ment** [comment]
Yes, I know this response come 2.5 years to late. The repo below might solve your problem. 
https://github.com/riskymanag3ment/Powershell4Tenable
